README.md

```diff
@@ -84,13 +84,24 @@ $ cabal build curiosity
 $ ./dist-newstyle/build/x86_64-linux/ghc-8.6.5/curiosity-0.1.0.0/x/curiosity/build/curiosity/cty serve
 ```
 
-And finally, we can build a binary with Nix:
+We can build a binary with Nix:
 
 ```
 $ nix-build -A curiosity
 $ ./result/bin/cty serve
 ```
 
+And finally, we can run a local dev environment containing curiosity and a Nginx reverse proxy using:
+
+```
+$ nix-build -A integration-tests.interactiveIntegrationEnv
```

Comment:
This is great, I like how it mimics the `nix-build -A runvm ; result/bin/run-nixos-vm` invocation.

I think this shouldn't be called "integration" or "tests". Just like the above `runvm` or `run-nixos-vm` are not specifically tied to the concept of testing or staging or production environment, I think this part of your work is "just" about running the Curiosity stack locally, directly on the host; it is "just" an env.

So maybe we could call the Nix attribute `run` and the script something like also `run` or `run-procfile` ?

Reply:
Renamed to `run.run-full-environment`.